docs/ops/arithmetic/Round_5.md

```diff
@@ -0,0 +1,52 @@
+## Round <a name="Round"></a> {#openvino_docs_ops_arithmetic_Round_5}
+
+**Versioned name**: *Round-5*
+
+**Category**: Arithmetic unary operation 
+
+**Short description**: *Round* performs element-wise round operation with given tensor. It takes one input tensor and rounds the values, element-wise, meaning it finds the nearest integer for each value. In case of halfs, the rule is to round them to the nearest even integer.
+
+**Attributes**:
+
+    No attributes available.
+
+**Inputs**
+
+* **1**: An tensor of type T. **Required.**
+
+**Outputs**
+
+* **1**: The result of element-wise round operation. A tensor of type T.
+
+**Types**
+
+* *T*: any numeric type.
+
+*Round* does the following with the input tensor *a*:
+
+\f[
+a_{i} = round(a_{i})
+\f]
```

Comment:
It is not clear what it actually does. Also missing the section header.

Reply:
Maybe is better way to remove this section?

Reply:
Agree.

Reply:
Ok, removed.